docker-entrypoint.sh

```diff
@@ -2,29 +2,61 @@
 set -e
 
 export GC_DOMAIN="${RENDER_EXTERNAL_HOSTNAME}"
+export DB_HOSTNAME=${DB_HOSTNAME:-'oregon-postgres.render.com'}
+
+migrate_db ()
+{
+  goatcounter migrate \
+    -createdb \
+    -db "$DB_CONNECTION_STRING" \
+    all
+}
+
+create_site ()
+{
+  goatcounter create \
+    -domain "$GC_DOMAIN" \
+    -email "$GC_USER_EMAIL" \
+    -password "$GC_PASSWORD" \
+    -db "$DB_CONNECTION_STRING"
+}
+
+{
+  migrate_db
+} || {
+  echo 'Failed to run migrations'
```

Comment:
We want to exit after these failures right? `set -o pipefail` at the top should do that I believe

Reply:
We don't want to exit after these two failures, these only show we have a site already created and no migrations to run

Reply:
got it! also I was wrong about pipefail, it shouldn't affect this